google_compute_engine/networking/network_daemon.py

```diff
@@ -58,26 +58,19 @@ def __init__(
     facility = logging.handlers.SysLogHandler.LOG_DAEMON
     self.logger = logger.Logger(
         name='google-networking', debug=debug, facility=facility)
-    self.network_utils = network_utils.NetworkUtils(logger=self.logger)
-    self.watcher = metadata_watcher.MetadataWatcher(logger=self.logger)
-
+    self.dhclient_script = dhclient_script
+    self.dhcp_command = dhcp_command
     self.ip_aliases = ip_aliases
+    self.ip_forwarding_enabled = ip_forwarding_enabled
+    self.network_setup_enabled = network_setup_enabled
     self.target_instance_ips = target_instance_ips
+    self.debug = debug
```

Comment:
Some of these statements are no longer needed.

Reply:
Thanks!